substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -912,22 +913,29 @@ impl PalletCmd {
 	}
 
 	/// Parses the PoV modes per benchmark that were specified by the `#[pov_mode]` attribute.
-	fn parse_pov_modes(benchmarks: &Vec<SelectedBenchmark>) -> Result<PovModesMap> {
+	fn parse_pov_modes(
```

Comment:
@ggwpez just out of curiosity, for example where is the  `#[pov_mode]` used?

Reply:
It is used extensively in the contracts pallet and a few other spots where there are no `MaxEncodedLen` bounds possible.  

https://github.com/paritytech/polkadot-sdk/blob/cb9fdb57faf02a7de3c2552f241e4e100ebfdcf8/substrate/frame/contracts/src/benchmarking/mod.rs#L241

Reply:
ahaaa, I see, `#[benchmark(pov_mode = Measured)]`, I was searching for `#[pov_mode]`,
maybe we could change docs here from `#[pov_mode]` -> `#[benchmark(.., pov_mode = <>` or something